backend/src/main/java/com/zpi/backend/game_instance/GameInstanceService.java

```diff
@@ -85,6 +90,8 @@ public void deactivate(String gameInstanceUUID, Authentication authentication) t
         GameInstance gameInstance = gameInstanceOptional.get();
         if (!gameInstance.isActive())
             throw new GameInstanceStatusException("The Game Instance (uuid = "+gameInstanceUUID+") has already been deactivated");
+//        Setting all future reservations as CANCELED BY OWNER
+        reservationRepository.setReservationStatusAsCanceledByOwnerForGameInstance(gameInstance);
```

Comment:
we probably have a method in reservationService to do this (unless it will cause this circular depencency)

Reply:
<img width="668" alt="Screenshot 2023-11-26 at 21 45 39" src="https://github.com/zpi-project/gameshare-app/assets/91901843/dbf9b5bb-7871-4cc5-888b-e65aa4754a1e">
Yup, in ReservationService we already have GameInstanceService.

Reply:
+ You used reservationRepository in other methods in GameInstanceService